content/docs/1.6.0/deploy/important-notes/index.md

```diff
@@ -106,3 +106,26 @@ Starting from `v1.5.0`, the following deprecated custom resource fields will be
 - BackingImageManager.status.backingImageFileMap.directory
 - BackingImageManager.status.backingImageFileMap.downloadProgress
 - BackingImageManager.status.backingImageFileMap.url
+
+### Engine Upgrade Enforcement
+
+Starting from `v1.6.0`, Longhorn will implement mandatory version upgrades for the engines. The minimum required supporting version will be outlined in the [release note](https://github.com/longhorn/longhorn/releases/tag/v{{< current-version >}}).
+
+When performing a helm upgrade, incompatible versions are automatically detected and prevented through a pre-hook mechanism. However, if you've installed Longhorn using manifests, the enforcement is handled by the longhorn-manager. This will result in unsuccessful pod launches and corresponding error logs. If you encounter such an error, it's necessary to revert to the previous version and upgrade any engine currently utilizing the incompatible engine image.
+
+You can assess the engine image versions for your current engines by utilizing the script below:
```

Comment:
You can determine the versions of engine images that are currently in use with the following script: